persistence/jdbc/src/main/java/org/infinispan/persistence/jdbc/impl/table/AbstractTableManager.java

```diff
@@ -294,18 +298,36 @@ protected String initSelectIdRowSql() {
       return String.format("SELECT %s FROM %s WHERE %s = ?", config.idColumnName(), tableName, config.idColumnName());
    }
 
+   protected String initSelectIdRowSqlWithLike() {
+      return String.format("SELECT %s FROM %s WHERE %s LIKE ?", config.idColumnName(), tableName, config.idColumnName());
+   }
+
    @Override
    public String getSelectIdRowSql() {
       return selectIdRowSql;
    }
 
+   @Override
+   public String getSelectIdRowSqlWithLike() {
+      return selectIdRowSqlWithLike;
+   }
+
    protected String initCountNonExpiredRowsSql() {
       return "SELECT COUNT(*) FROM " + tableName +
             " WHERE " + config.timestampColumnName() + " < 0 OR " + config.timestampColumnName() + " > ?";
    }
 
+   protected String initCountRowsSql() {
+      return "SELECT COUNT(*) FROM " + tableName;
+   }
+
    @Override
    public String getCountNonExpiredRowsSql() {
+      return countNonExpiredRowsSql;
+   }
+
+   @Override
+   public String getCountRowsSql() {
```

Comment:
Ideally we shouldn't be adding SQL to the `src/main` files just for use in test cases. This SQL is part of the standard, so it should run on all DBs, i.e. no need to override this SQL on a per DB level, so I think you should just use this directly in your test `TableManipulation` class.

Reply:
@ryanemerson updated